Custom init.sh script, master/admin passwords via ENV vars

OndroMih opened the pull request:
Read changes in content.md for description.

Fixes https://github.com/eclipse-ee4j/glassfish.docker/issues/9.

In short:

The following environment variables can be set to configure GlassFish Server:

* `AS_ADMIN_MASTERPASSWORD` - to change the default master password
* `AS_ADMIN_PASSWORD` - to change the default admin password

It's possible to specify custom commands to run before GlassFish server starts. Just create a Bash script `${PATH_GF_HOME}/custom/init.sh` (`/opt/glassfish7/custom/init.sh`), it will be executed before GlassFish server starts.

OndroMih committed:
init.sh and init.asadmin, master/admin passwords via ENV vars

Read content.md for description

OndroMih committed:
Add test for changing admin password.

Make /password.txt writable so that it can be changed when admin password changes.

OndroMih committed:
Improve the procedure of changing passwords

Change both admin and master password in one asadmin execution.

dmatej commented on src/test/java/org/glassfish/main/distributions/docker/StartServIT.java:
```diff
@@ -16,47 +16,24 @@
     @SuppressWarnings({"rawtypes", "resource"})
     @Container
     private final GenericContainer server = new GenericContainer<>(System.getProperty("docker.glassfish.image"))
-        .withCommand("startserv").withExposedPorts(8080)
```
Why you removed this? I would rather like to see it explicitly here (is it default?)

dmatej commented on test/docker-entrypoint_.sh:
```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```
File name with underscore before the suffix is suspicious, can it have a better name?

dmatej commented on src/test/java/org/glassfish/main/distributions/docker/CommonIntegrationTests.java:
```diff
@@ -0,0 +1,65 @@
+ *
+ * @author Ondro Mihalyi
+ */
+public final class CommonIntegrationTests {
```
Maybe better names ... or ... keep assertions in tests, and move away just the download code. And create a Record for `HttpResponse` with two fields, `code` and `body`.
Then this class could have a name like `HttpUtilities` instead of hiding tests so I have to open it to see what we are really checking.

OndroMih committed:
Fixes and refactors

Remove file with underscore and apply changes to the original file.
Refactor tests - use Java Http client instead of HttpURLConnection.
Bring back startserv test, create a new test for default command.
